src/formApps/_ActiveEffects/components/category/CategoryControl.svelte

```diff
@@ -69,8 +69,9 @@
   }
 
   //let itemName = item.label;
+  const realName = AAAutorecFunctions.getRealName($animation.label, item);
   //Check the Autorec Menu for a matching Section
-  $: isInAutorec = AAAutorecFunctions.singleMenuSearch(aefxMenu, AAAutorecFunctions.rinseName($animation.label), $animation.label);
+  $: isInAutorec = AAAutorecFunctions.singleMenuSearch(aefxMenu, AAAutorecFunctions.rinseName(realName), realName);
```

Comment:
This will likely never update due to realName being a const. This is a svelte file, with reactivity being key.